torch/_inductor/codegen/triton.py

```diff
@@ -1050,45 +1050,25 @@ def mask_loads(self, mask):
         finally:
             self._load_mask = prior
 
-    def gen_assert_indirect_indexing(self, buffer, original_index, mask):
-        if mask == "None":
-            return
-        body = self.current_node._body
-        indirect_size = dict(zip(body.indirect_vars, body.indirect_max_sizes))
```

Comment:
You can then also revert the code I added that populates ìndirect_max_sizes` and similar structures.

Reply:
I spotted a problem with this. Theoretically you can have one maximum under one set of masks, and another under a different set of masks. Not an insurmountable problem, but I suspect it wouldn't be worth the complexity.